src/backends/interflop-mca/interflop_mca.c

```diff
@@ -186,6 +189,16 @@ static double _mca_rand(void) {
   return tinymt64_generate_doubleOO(&random_state);
 }
 
+static bool _mca_skip_eval(const int sparsity) {
```

Comment:
inline ?

Reply:
Hi @titeup , I'm not sure I understand what you mean here. Sorry about that.

Reply:
static inline bool ...
To instruct the compiler to inline your call and avoid call overhead. This is useful since backends are not using aggressive compiler options to not mess the very sensitive (and difficult to debug) arithmetic we are using.

Reply:
```suggestion
static inline bool _mca_skip_eval(const int sparsity) {
```

Reply:
Thanks!